src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx

```diff
@@ -60,76 +64,96 @@ const TodoItem = createReactClass({
     this.props.onSkip(task);
     this.setState({showConfirmation: false});
   },
 
-  render: function() {
+  handleClick(e) {
+    this.recordAnalytics('clickthrough');
+    e.stopPropagation();
+  },
+
+  render() {
+    let {task, className} = this.props;
+    let {showConfirmation} = this.state;
     let learnMoreUrl = this.learnMoreUrlCreator();
     let description;
 
-    switch (this.props.task.status) {
+    switch (task.status) {
       case 'complete':
         description = tct('[user] completed [dateCompleted]', {
-          user: this.props.task.user,
-          dateCompleted: moment(this.props.task.dateCompleted).fromNow(),
+          user: task.user,
+          dateCompleted: moment(task.dateCompleted).fromNow(),
         });
         break;
       case 'pending':
         description = tct('[user] kicked off [dateCompleted]', {
-          user: this.props.task.user,
-          dateCompleted: moment(this.props.task.dateCompleted).fromNow(),
+          user: task.user,
+          dateCompleted: moment(task.dateCompleted).fromNow(),
         });
         break;
       case 'skipped':
         description = tct('[user] skipped [dateCompleted]', {
-          user: this.props.task.user,
-          dateCompleted: moment(this.props.task.dateCompleted).fromNow(),
+          user: task.user,
+          dateCompleted: moment(task.dateCompleted).fromNow(),
         });
         break;
       default:
         description = this.formatDescription();
     }
 
-    let classes = classNames(this.props.className, this.props.task.status, {
-      blur: this.state.showConfirmation,
+    let classes = classNames(className, task.status, {
+      blur: showConfirmation,
     });
 
     let showSkipButton =
-      this.props.task.skippable &&
-      this.props.task.status != 'skipped' &&
-      this.props.task.status != 'complete' &&
-      !this.state.showConfirmation;
+      task.skippable &&
+      task.status != 'skipped' &&
+      task.status != 'complete' &&
+      !showConfirmation;
 
     return (
       <li
         className={classes}
         onMouseOver={this.toggleDescription}
         onMouseOut={this.toggleDescription}
       >
-        {this.props.task.status == 'pending' && <div className="pending-bar" />}
+        {task.status == 'pending' && <div className="pending-bar" />}
         <div className="todo-content">
           <div className="ob-checkbox">
-            {this.props.task.status == 'complete' && <span className="icon-checkmark" />}
-            {this.props.task.status == 'skipped' && <span className="icon-x" />}
-            {this.props.task.status == 'pending' && <span className="icon-ellipsis" />}
+            {task.status == 'complete' && <span className="icon-checkmark" />}
+            {task.status == 'skipped' && <span className="icon-x" />}
+            {task.status == 'pending' && <span className="icon-ellipsis" />}
+          </div>
+          <div onClick={this.handleClick}>
+            <a href={learnMoreUrl}>
```

Comment:
I haven't checked out the branch and tested, but would this be better if it was an `ExternalLink` so that it opens in a new tab?

Also, is it possible to apply `onClick` on `<a>` rather than adding a wrapper `div`?

Reply:
Yes to moving to `<a>` - i'd played around with including the description in the `onclick` div for a second.

For the `ExternalLink` for half the links it makes sense (when it points to docs for example) but for others like create a project it makes sense to keep them on the same page. I think it's worse to have those pop up in another tab than to have the current experience. Thoughts?

Reply:
Cool, I didn't have context with where the links are going, so that makes sense not to have a new tab.